src/components/Search.js

```diff
@@ -6,26 +6,29 @@ class Search extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      // default placesholders while page is loading
-      location: 'Enter Location',
-      date: 'mm/dd/yyyy',
+      location: 'Enter a location',
     };
   }
 
-  componentDidMount() {
-    // sets placeholder values of the date & location fields to curr loc and curr date
-    const today = getdate('mm/dd/yyyy');
-    const _this = this;
```

Comment:
Got rid of contextual placeholder -- because arrow function maintains 'this' binding?